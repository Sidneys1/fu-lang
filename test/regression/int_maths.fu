main: u32() = {
  a: u32 = 100;
  b: u32 = 2;
  c: u32 = a - b; // 98
  d: u32 = a * b; // 200
  e: u32 = a / b; // 50
  return c + d + e; // 98 + 200 + 50 = 348
};
