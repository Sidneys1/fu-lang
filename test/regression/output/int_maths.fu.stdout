----------- test\regression\int_maths.fu -----------
[2m   4 |   c: u32 = [0;2ma - b;[2m // 98[0m[0;0m <-- Checks for infix operator '-' are not implemented! [0;2m(test\regression\int_maths.fu:4:12-17)
[0m[m[0;2m-----+[0m
[2m   5 |   d: u32 = [0;2ma * b;[2m // 200[0m[0;0m <-- Checks for infix operator '*' are not implemented! [0;2m(test\regression\int_maths.fu:5:12-17)
[0m[m[0;2m-----+[0m
[2m   6 |   e: u32 = [0;2ma / b;[2m // 50[0m[0;0m <-- Checks for infix operator '/' are not implemented! [0;2m(test\regression\int_maths.fu:6:12-17)
[0m[m[0;2m-----+[0m
[2m   7 |   return [0;2mc + d [2m+ e; // 98 + 200 + 50 = 348[0m[0;0m <-- Checks for infix operator '+' are not implemented! [0;2m(test\regression\int_maths.fu:7:10-15)
[0m[m[0;2m-----+[0m
[2m   7 |   return [0;2mc + d + e;[2m // 98 + 200 + 50 = 348[0m[0;0m <-- Checks for infix operator '+' are not implemented! [0;2m(test\regression\int_maths.fu:7:10-19)
[0m[m